docs/csharp/language-reference/keywords/ref.md

```diff
@@ -109,11 +109,11 @@ ref VeryLargeStruct reflocal = ref veryLargeStruct;
 
 Note that in both examples the `ref` keyword must be used in both places, or the compiler generates error CS8172, "Cannot initialize a by-reference variable with a value."
 
-Beginning with C# 7.3, the iteration variable of the `foreach` statement can be ref local or ref readonly local variable. For more information, see the [foreach statement](foreach-in.md) article.
+Beginning with C# 7.3, the iteration variable of the `foreach` statement can be ref local or readonly ref local variable. For more information, see the [foreach statement](foreach-in.md) article.
 
-## Ref readonly locals
+## Readonly ref locals
 
-A ref readonly local is used to refer to values returned by the method or property that has `ref readonly` in its signature and uses `return ref`. A `ref readonly` variable combines the properties of a `ref` local variable with a `readonly` variable: it is an alias to the storage it's assigned to, and it cannot be modified. 
+A readonly ref local is used to refer to values returned by the method or property that has `readonly ref` in its signature and uses `return ref`. A `readonly ref` variable combines the properties of a `ref` local variable with a `readonly` variable: it is an alias to the storage it's assigned to, and it cannot be modified. 
```

Comment:
For local variables and return types, `ref readonly` is the right ordering. [See here at SharpLab.](https://sharplab.io/#v2:C4LgbgpgTglgZgTwD4AEBMBGAsAKF+gAgGFcBvXAyglAFgIFkAKGAO2AJgEoKrycqBBKBDhCIAQwAmAexYAbBBzZCMBALxjRMANw9BYqbIWal7KGnUmdeggF8bN4aOGH5i1u3oZGndQD4CYAALKGkAdwIWCAiAUQAPAGMIAAdgGFkfXX4qFxk3K2V6NB9/QJDwyOiCeKTU9JZM3FsgA=) It's only `struct`s that are `readonly ref`.

 So most of these changes should be reverted.